docs/dev/notebooks.md

```diff
@@ -48,19 +48,34 @@ Thus, for notebooks with external dependencies, **all cells must have their outp
 
 ## Lifecycle 
 
-We should follow different configurations for notebooks depending on whether they rely on features in the pre-release build of cirq or not. 
+You should configure notebooks differently depending on whether they rely on features in the pre-release build of cirq or not. 
 
-Pre-release notebooks: 
+### Pre-release notebooks
+
+When you introduce a notebook that depends on pre-release features of Cirq, make sure to 
+ 
  - mark the notebook at the top that `Note: this notebook relies on unreleased Cirq features. If you want to try these feature, make sure you install cirq via pip install cirq --pre`. 
  - use `pip install cirq —pre`  in the installation instructions 
  - make sure dev_tools/notebooks/test_notebooks.py covers the notebook 
  - exclude the notebook from the dev_tools/notebooks/isolated_notebook_test.py by adding it to `NOTEBOOKS_DEPENDING_ON_UNRELEASED_FEATURES`
 
-After the Cirq release - for all unreleased notebooks we change all the above accordingly in bulk for all the notebooks: 
+### Stable notebooks
+
+When you introduce a notebook that only uses already released features of Cirq, make sure to
+ - use `pip install cirq` (NOT `pip install cirq --pre`)
+ - do not exclude the notebook from dev_tools/notebooks/isolated_notebook_test.py (except if the notebook has external dependencies)
+ - do not exclude the notebook dev_tools/notebooks/notebook_test.py (except if the notebook has external dependencies)
```

Comment:
duplicated line?

Reply:
it is not - the file names are different! :) Maybe I could merge them if that's not obvious